openreview/conference/matching.py

```diff
@@ -769,9 +769,9 @@ def _build_config_invitation(self, scores_specification):
                 'signatures': {'values': [self.conference.get_id()]},
                 'content': {
                     'title': {
-                        'value-regex': '.{1,250}',
+                        'value-regex': '(?!^ +$)^[a-zA-Z0-9-_ ]{1,250}$',
```

Comment:
what is the negative lookahead here for?

Reply:
To check for only spaces.

Reply:
I see, I think it would be better to have a regex like this instead:
```
^[a-zA-Z0-9-_][a-zA-Z0-9-_ ]{0,250}$
```
This achieves the same thing in a simpler way. It also disallows spaces at the beginning.